s12/protobuf/workato/annotations.pb.go

```diff
@@ -1,6 +1,6 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.36.4
+// 	protoc-gen-go v1.34.1
```

Comment:
version going down?

Reply:
??? not sure, that is autogenerated. It is a file for tests